pythonforandroid/distribution.py

```diff
@@ -60,6 +67,13 @@ def get_distribution(cls, ctx, name=None, recipes=[],
         name : str
             The name of the distribution. If a dist with this name already '
             exists, it will be used.
+        ndk_api : int
+            The NDK API to compile against, included in the dist because it cannot
+            be changed later during APK packaging.
+        arch_name : str
+            The target architecture name to compile against, included in the dist because
+            it cannot be changed later during APK packaging.
+            This is a required, keyword-only argument.
```

Comment:
I'm tempted to say this goes without saying `This is a required, keyword-only argument`. Specifically `keyword-only argument` is the case for all except for `cls`

Reply:
Fair, will change.